ios/Podfile.lock

```diff
@@ -117,7 +117,7 @@ EXTERNAL SOURCES:
 
 SPEC CHECKSUMS:
   device_info_plus: c6fb39579d0f423935b0c9ce7ee2f44b71b9fce6
-  Flutter: f04841e97a9d0b0a8025694d0796dd46242b2854
+  Flutter: e0871f40cf51350855a761d2e70bf5af5b9b5de7
```

Comment:
looks like you're using a different flutter version.. what do you get when running `which flutter`? it should point to `didpay/bin/flutter`

if the above is correct, then `flutter --version` should give you Flutter 3.19.3

Reply:
@ethan-tbd 

```
flutter --version
Flutter 3.19.3 • channel stable • https://github.com/flutter/flutter.git
Framework • revision ba39319843 (2 weeks ago) • 2024-03-07 15:22:21 -0600
Engine • revision 2e4ba9c6fb
Tools • Dart 3.3.1 • DevTools 2.31.1
which flutter
/Users/micn/Documents/code/didpay/bin/flutter
```